django/db/backends/base/validation.py

```diff
@@ -7,4 +7,20 @@ def check(self, **kwargs):
         return []
 
     def check_field(self, field, **kwargs):
+        if hasattr(self, 'check_field_type') and self.can_check_field_db_type(field):
+            field_type = field.db_type(self.connection)
+            # Ignore non-concrete fields.
+            if field_type is not None:
+                return self.check_field_type(field, field_type)
         return []
```

Comment:
I guess this should use the same `errors = [] ... errors.append() ... return errors` approach that similar methods use?